### DISCUSSION TITLE
Getting MODIS Data

### DISCUSSION THREAD

**Discussion by jejjohnson** [spaceml-org/rs_tools#2] [General]
Below are some notes or ways that one can acquire MODIS data.

**Google Earth Engine**. There are a few readily available 

* [gee-tool](https://github.com/gee-community/gee_tools)
* [wxee](https://wxee.readthedocs.io/en/latest/) - xarray-based

**Other Tools**

* [modis-tool](https://github.com/fraymio/modis-tools)

**Comment by nkasmanoff** [comment]
Hey @jejjohnson thank you for adding these discussions, reading through them now. Not sure if this is where I should add these thoughts, happy to delete or discuss somewhere else. If of interest I can try and add the answers to them. 

What are the spectral bands of Modis & it's resolution? Max AoI size? 

How long it takes to download an image or set of images this way? 

Is there a "cutoff" date for when the images aren't available? 

Can you pre-filter based on cloud coverage?

**Reply by jejjohnson** [reply]
Thanks for the discussion! So...I realized this isn't a good place for building a knowledge base. Instead, I've added stuff to the wiki! So yeah, anyone can start to this important information directly there! Sorry for the confusion. I'm still trying to figure out the workflow...
